doc/index.rst

```diff
@@ -390,6 +390,14 @@ In such cases, you can specify the script(s) in `BEFORE_SCRIPT` and/or `AFTER_SC
   script:
     - .ci_config/travis.sh
 
+Multiple commands can be passed, as in a general `bash` manner. Using semi-colon as a delimitter,::
+
+    - BEFORE_SCRIPT="ls /tmp/1; ls /tmp/2"
```

Comment:
Please use single quotes in these examples. So users won't copy error-prone settings.